app/assets/javascripts/bootstrap.js.coffee

```diff
@@ -1,11 +1,3 @@
-#
-#= require twitter/bootstrap
-#= require datetimepicker/bootstrap-datetimepicker
-#= require bootstrap-fileupload
-#= require bootstrap-multiselect
```

Comment:
This gets added again in the next commit. It would be good to rebase such reverted changes into the previous commit.